tools/sof-tplgreader.py

```diff
@@ -46,6 +59,25 @@ def loadFile(self, filename, sofcard=0):
                 pipeline_dict['fmt'] = pipeline_dict['fmts'].split(' ')[0]
                 pipeline_dict['rate_min'], pipeline_dict['rate_max'] = self._key2str(cap, 'rate')
                 pipeline_dict['ch_min'], pipeline_dict['ch_max'] = self._key2str(cap, 'channels')
+                if pcm_type == "both":
+                    pipeline_dict["type"] = "capture"
+                    # copy pipeline and change values
+                    pb_pipeline_dict = copy.copy(pipeline_dict)
```

Comment:
I saw you rebuild the 'pga' and 'eq' don't need deep copy, you can direct to use `pipeline_dict.copy`

Reply:
sure, will refine in next push